downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -613,6 +613,11 @@ $= future<int> result = start name();
 
 #### CLI
 
+#### EDI tool
+
+- Introduced support for field length constraints (min/max), enhancing validation capabilities for EDI data fields.
+- Added support for EDIFACT to Ballerina schema conversion, facilitating smoother integrations and automations.
```

Comment:
Shall we give priority to this feature and move it as first point?

Reply:
Changed with [ae1fe62](https://github.com/ballerina-platform/ballerina-dev-website/pull/8938/commits/ae1fe629fdfda02b5bb14665929c822be4a9c69e).